Utilizr.WPF/Controls/ClickableRun.cs

```diff
@@ -46,6 +46,15 @@ public ClickableRun()
                     Command?.Execute(CommandParameter);
                 }
             };
+
+            MouseLeftButtonDown += (s, e) =>
+            {
+                if (Command?.CanExecute(CommandParameter) == true)
+                {
+                    // Null check, as unlikely, but possibly changed can execute check
+                    Command?.Execute(CommandParameter);
+                }
```

Comment:
This will mean the `Command` can fire twice. We'll need to add a `MouseDownCommand` equivalent for `Command`, and the same for `MouseDownCommandParameter` for `CommandParameter`. Then consumers can decide which one they can use.